<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Order</class>
 <widget class="QDialog" name="Order">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>1181</width>
     <height>721</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Perpetua Titling MT</family>
     <pointsize>9</pointsize>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>ORDER </string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>   Order List</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="foods_groupBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>40</y>
      <width>551</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Foods</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>501</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QPushButton" name="product1_pushButton">
        <property name="font">
         <font>
          <family>Perpetua Titling MT</family>
          <pointsize>9</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: rgb(255, 255, 255);
}</string>
        </property>
        <property name="text">
         <string>Brownie</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="product2_pushButton">
        <property name="text">
         <string>Sandwich</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="product3_pushButton">
        <property name="text">
         <string>Cheesecake</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="product4_pushButton">
        <property name="text">
         <string>Macaron</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="product5_pushButton">
        <property name="text">
         <string>Taco</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="product6_pushButton">
        <property name="text">
         <string>Croissant</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="hotdrinks_groupBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>250</y>
      <width>551</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Hot Drinks</string>
    </property>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>507</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="product7_pushButton">
        <property name="font">
         <font>
          <family>Perpetua Titling MT</family>
          <pointsize>9</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>FLAT WHITE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="product8_pushButton">
        <property name="text">
         <string>Latte</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="product9_pushButton">
        <property name="text">
         <string>Americano</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="product10_pushButton">
        <property name="text">
         <string>Espresso</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="product11_pushButton">
        <property name="text">
         <string>Cappuccino</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="product12_pushButton">
        <property name="text">
         <string>Mocha</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="product13_pushButton">
        <property name="text">
         <string>Tea</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="colddrinks_groupBox">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>440</y>
      <width>551</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Perpetua Titling MT</family>
      <pointsize>9</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Cold Drinks</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>511</width>
       <height>101</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="product14_pushButton">
        <property name="text">
         <string>Mojito</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="product15_pushButton">
        <property name="text">
         <string>Coke</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="product16_pushButton">
        <property name="text">
         <string>Lemonade</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="product17_pushButton">
        <property name="text">
         <string>Ice Tea</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="product18_pushButton">
        <property name="text">
         <string>Soda</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="product19_pushButton">
        <property name="text">
         <string>Smoothies</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>product14_pushButton</zorder>
     <zorder>product15_pushButton</zorder>
     <zorder>product17_pushButton</zorder>
     <zorder>product18_pushButton</zorder>
     <zorder>product19_pushButton</zorder>
     <zorder>product16_pushButton</zorder>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>430</y>
      <width>371</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="submitOrder_pushButton">
       <property name="text">
        <string>SUBMIT ORDER</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="calculate_pushButton">
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="orderlist_groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>501</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string>    PRODUCT                        PRICE                      AMOUNT</string>
    </property>
    <widget class="QListWidget" name="listWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>481</width>
       <height>281</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Perpetua Titling MT</family>
       <pointsize>8</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>480</y>
      <width>371</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Payment</string>
    </property>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>80</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="cashPayment_pushButton">
        <property name="font">
         <font>
          <family>Perpetua Titling MT</family>
          <pointsize>9</pointsize>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cash</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="creditCardPayment_pushButton">
        <property name="text">
         <string>Credit Card</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Perpetua Titling MT</family>
       <pointsize>9</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TOTAL BILL: </string>
     </property>
    </widget>
    <widget class="QListWidget" name="totalBill_listWidget">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>670</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="increase_pushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>490</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="decrease_pushButton">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>530</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QListWidget" name="TableNo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
